lib/blockscore/errors/blockscore_error.rb

```diff
@@ -0,0 +1,20 @@
+module BlockScore
+  class BlockScoreError < StandardError
+    attr_reader :message
+    attr_reader :http_status
+    attr_reader :http_body
+    attr_reader :json_body
+
+    def initialize(message=nil, http_status=nil, http_body=nil, json_body=nil)
```

Comment:
From discussion: Doesn't really make sense to have four optional params, especially when inheriting classes screw up the ordering. A hash would make more sense